atlantis-aws/variant.mod

```diff
@@ -26,6 +32,21 @@ dependencies:
       githubReleases:
         source: runatlantis/atlantis
     version: "> 0.0.0"
+  dumb_init:
+    releasesFrom:
+      githubReleases:
+        source: Yelp/dumb-init
+    version: "> 0.0.0"
+  gosu:
+    releasesFrom:
+      githubReleases:
+        source: tianon/gosu
+    version: "> 0.0.0"
+  git_lfs:
+    releasesFrom:
+      githubReleases:
+        source: git-lfs/git-lfs
+    version: "> 0.0.0"
```

Comment:
@cw-sakamoto Thanks. Since dumb-init is pinned in atlantis base image, only dumb-init version is fixed.
https://github.com/runatlantis/atlantis/pull/2587